keyboards/mini49/keymaps/vial/config.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0x5C, 0x8A, 0x21, 0xB3, 0x82, 0x40, 0xF6, 0x11}
+
+#define VIAL_UNLOCK_COMBO_ROWS {0}
+#define VIAL_UNLOCK_COMBO_COLS {0}
```

Comment:
Per [the docs](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination):
> You should configure a combo of at least 2 keys

This may not be a strict requirement for boards with more than one key. I'd still adhere to it.
```suggestion
#define VIAL_UNLOCK_COMBO_ROWS {0, 0}
#define VIAL_UNLOCK_COMBO_COLS {0, 9}
```

Reply:
Added suggested.